src/js/components/List/doc.js

```diff
@@ -171,4 +189,10 @@ export const themeDoc = {
     type: 'string | (props) => {}',
     defaultValue: undefined,
   },
+  'list.pagination': {
+    description: `If paginating, any valid Box props to style the pagination 
+    component.`,
+    type: 'object',
+    defaultValue: `margin: { top: 'xsmall' }`,
```

Comment:
I don't see this default value in base.js, I think we achieve the desired styling with list.container. Can this be removed?

Reply:
Fixed.